web-api/setup-elasticsearch-index.sh

```diff
@@ -20,29 +20,26 @@
 
```

Comment:
Not sure why this file is different, this isn't apart of my pr, but don't want to break test, please confirm this is OK before merging to test.

Reply:
This is an update for our migration process, Mike worked on this